### PR TITLE
Draft pull request not ready for review now

### PR DESCRIPTION
**Pull request by geronimoEKIA**
Draft pull request not ready for review now

### CODE REVIEW

File: hungry.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 hungry==input("are you hungry?")
-if hungry=="yes":
+if hungry=="yes" and hungry=="y":
```
Comment (harsh4870) [inline hungry.py]: @geronimoEKIA  Can you please add Y as answer ?

Comment (geronimoEKIA) [reply hungry.py]: Sure i have updated code

Comment (geronimoEKIA) [reply hungry.py]: @harsh4870  you can check code changes

Comment (harsh4870) [reply hungry.py]: Done thanks

**Pull request merged**